packages/guides-restructured-text/src/RestructuredText/Parser/Productions/DirectiveRule.php

```diff
@@ -184,20 +184,19 @@ private function parseDirectiveOption(string $line): DirectiveOption
     private function collectDirectiveContents(LinesIterator $documentIterator): Buffer
     {
         $buffer = new Buffer();
-        $indenting = 1;
-        while (LinesIterator::isBlockLine($documentIterator->getNextLine(), $indenting)) {
+        $minIndenting = 1000;
```

Comment:
Could you please extract the default value to a constant, so we can add docblocks above it to explain what it is? Right now it feels like a random number.

Reply:
It is a random number it just needs to be very high...

Reply:
Mmh, maybe with PHP_INT_MAX ?